### ISSUE TITLE
Inventory command to bootstrap a manifest file for the Impact Framework

### ISSUE THREAD

**Issue by jnioche** [Boavizta/cloud-scanner#462]
The Impact Framework from the Green Software Foundation takes as input a manifest file. This is a yaml file listing the resources (e.g. EC2 instances) and related observations (CPU usage etc...). Given that this is pretty much what we get from the inventory command on the CLI, it would make it easier to bootstrap a manifest file.

The Impact Framework has a plugin for the Boavizta API so the results should be the same as calling the `estimate` command.

**Comment by jnioche** [comment]
This could of course also be done as a plugin for the Impact Framework instead. There is something similar for [Azure](https://github.com/Green-Software-Foundation/if-unofficial-plugins/blob/main/src/lib/azure-importer/README.md) already but not for AWS

**Comment by demeringo** [comment]
Yes this would be interresting to see if we can generate this manifest file.

We have add this feature in cloud scanner backlog:  to be able to generate an inventory as file and be able to replay it to get impacts https://github.com/Boavizta/cloud-scanner/issues/352

**Comment by demeringo** [comment]
Format of the manifest file: https://if.greensoftware.foundation/major-concepts/manifest-file/

Not sure if there is an associated json schema.
